pkg/shp/cmd/buildrun/logs_test.go

```diff
@@ -178,7 +178,7 @@ func TestStreamBuildRunFollowLogs(t *testing.T) {
 		cmd.Cmd().ExecuteC()
 		pm := genericclioptions.NewConfigFlags(true)
 		if len(test.to) > 0 {
-			pm.Timeout = &test.to
+			*pm.Timeout = test.to
```

Comment:
Me, tired, trying to do pointer arithmetic.
![image](https://github.com/shipwright-io/cli/assets/1979133/737dfb91-741b-47d6-a2dd-b6410634f806)

Reply:
This is what prompted the change:
```
> make sanity-check                                                                                                                                              2023-08-23-bump-dependencies
golangci-lint run
pkg/shp/cmd/buildrun/logs_test.go:181:17: G601: Implicit memory aliasing in for loop. (gosec)
			pm.Timeout = &test.to
			             ^
make: *** [sanity-check] Error 1

```